### ISSUE TITLE
Recommend different contrast checker

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-easy-checks#12]
[Contrast](http://w3c.github.io/EasyChecks/#contrast) section:

Checking with Firefox suggests the [Juicy Studio Accessibility Toolbar](https://addons.mozilla.org/en-US/firefox/addon/juicy-studio-accessibility-too/). I would propose recommending the [WCAG Contrast checker](https://addons.mozilla.org/en-us/firefox/addon/wcag-contrast-checker/) add on instead.

**Rationale:** They are both downloads, so both subject to the potential access restrictions that some might have. The Juicy Studio tool is less focussed as it includes functionality on ARIA, tables, etc, which may lead to more confusion. Most importantly, the WCAG Contrast checker tool provides a much better indication of failure and makes it significantly easier to identify what fails.

**Comment by AndrewArch** [comment]
If we're considering alternative downloads, the [Colour Contrast Analyser](https://www.paciellogroup.com/resources/contrastanalyser/) (CCA) from Paciello is worth consideration partly because of it's multiple language options and it's availability on OSX and Windows. The CCA also assists the selection of better contrast choices.

**Comment by dboudreau** [comment]
Andrew - While I agree with you about wanting to promote CCA, I thought I'd mention that it only provides assistance on selection of better contrast choices in the Windows version. The MacOS version, alas, is much simpler and does not provide all of the same features (see screenshot).

![Color contrast analyzer for MacOS, screenshot](https://www.dropbox.com/s/8lkk551m818esea/Screenshot%202016-10-17%2009.49.55.png)

**Comment by dboudreau** [comment]
I think it would be interesting to benchmark a few of these tools if we want to recommend any. I personally use CCA on a regular basis, or when I need to pick colors (love those color pickets), but I'll use Snook's as well, just because IT HAS this feature Andrew mentioned: https://snook.ca/technical/colour_contrast/colour.html#fg=33FF33,bg=333333

Plus, it's web based which is also nice.

**Comment by sdeschamps-orange** [comment]
What about providing several tools? This mitigates the platform/implementation problem, in passing. :wink:

**Comment by iadawn** [comment]
Closed as no longer relevant to new version of the resource
